### DISCUSSION TITLE
Support export command to follow the path relatively to the diagram file, not to the cwd.

### DISCUSSION THREAD

**Discussion by ThePlenkov** [likec4/likec4#1228] [Ideas]
Hi! 

I have a following script which I run in the root of my project

```
npx likec4 export png -o ./img
```

Currently it generates images in a root ./img folder of my project

What I expect is that command line somehow resolves ./img relatively to each diagram and update file there.

So may be we can introduce something like `--relative` or `--resolution=relative` 

What do you think?

**Comment by davydkov** [comment]
Hey @ThePlenkov 

You can get _kinda_ the same if you omit `-o` (images will be next to source files).  
For example:

<img alt="SCR-20241115-scvy" src="https://github.com/user-attachments/assets/d4287dbf-9f2a-4690-b3b3-5de45ac7bc3a">

<br/>

<br/>

Produces:  

![SCR-20241115-sfej](https://github.com/user-attachments/assets/263be9c4-b295-4cd2-b861-b554e84dd274)

Also, you can post-process with some script, like move every `${basedir}/*.png` to `${basedir}/img/*.png`
